Validators/InputDataValidation.cs

```diff
@@ -4,13 +4,13 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace TrainingApp2
+namespace TrainingApp2.Validators
 {
     public class InputDataValidation
     {
         public string GetStringValueFromConsole()
         {
-            for(; ; )
+            for (; ; )
```

Comment:
tu tez niech bedzie petla while(true)

Reply:
zmieniłem wszystkie for (; ; ) na while(true)